content/en/docs/concepts/overview/working-with-objects/field-selectors.md

```diff
@@ -32,6 +32,18 @@ kubectl get ingress --field-selector foo.bar=baz
 Error from server (BadRequest): Unable to find "ingresses" that match label selector "", field selector "foo.bar=baz": "foo.bar" is not a known field selector: only "metadata.name", "metadata.namespace"
 ```
 
+### List of supported fields
+
+| Kind       | Fields                                                                                                                                                                                                                                                          |
+| ---------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Pod        | `spec.nodeName`<br>`spec.restartPolicy`<br>`spec.schedulerName`<br>`spec.serviceAccountName`<br>`spec.hostNetwork`<br>`status.phase`<br>`status.podIP`<br>`status.nominatedNodeName`                                                                            |
+| Event      | `involvedObject.kind`<br>`involvedObject.namespace`<br>`involvedObject.name`<br>`involvedObject.uid`<br>`involvedObject.apiVersion`<br>`involvedObject.resourceVersion`<br>`involvedObject.fieldPath`<br>`reason`<br>`reportingComponent`<br>`source`<br>`type` |
+| Secret     | `type`                                                                                                                                                                                                                                                          |
+| Namespace  | `status.phase`                                                                                                                                                                                                                                                  |
+| ReplicaSet | `status.replicas`                                                                                                                                                                                                                                               |
+| Job        | `status.successful`                                                                                                                                                                                                                                             |
+| Node       | `spec.unschedulable`                                                                                                                                                                                                                                            |
+
```

Comment:
As part of CRD field selectors, I compiled a list of existing field-selector usages: https://github.com/kubernetes/enhancements/blob/master/keps/sig-api-machinery/4358-custom-resource-field-selectors/README.md#background-field-selection-as-it-exists-before-this-enhancement

These lists are VERY similar, I think CertificateSigningRequest, ReplcationController and maybe the 'name' selector on Namespace/PV/PVC (which is a mistake) are the only things I was expecting to see here as well.

Reply:
Thank you. I missed the RC and CSR selectors.
I was aware of the 'name' thing, but decided not to include it because it seemed like something that should not be used anymore. Should I add them regardless?

Reply:
I'm okay with omitting the name case since it's an mis-feature.